efficientdet/keras/train_lib_test.py

```diff
@@ -32,11 +32,13 @@ def test_display_callback(self):
     fake_jpeg = tf.image.encode_jpeg(fake_image)
     sample_image = 'ram://fake_image.jpg'
     tf.io.write_file(sample_image, fake_jpeg)
-    display_callback = train_lib.DisplayCallback(sample_image, config.model_dir)
+    display_callback = train_lib.DisplayCallback(sample_image,
+                                                 config.model_dir,
+                                                 update_freq=1)
     model = train_lib.EfficientDetNetTrain(config=config)
     model.build((1, 512, 512, 3))
     display_callback.set_model(model)
-    display_callback.on_epoch_end(0, {})
+    display_callback.on_train_batch_end(0, {})
```

Comment:
I believe you can just use display_callback.on_train_batch_end(batch=0) ?

logs={} should be equivalent to logs=None (default)